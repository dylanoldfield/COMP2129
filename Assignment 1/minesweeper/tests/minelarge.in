g 6 6
b 1 1
b 1 2
b 1 3
b 2 1
b 2 3
b 3 1
b 3 2
b 3 3
b 4 1
b 4 2
u 0 0 
u 1 0
u 2 0
u 3 0
u 4 0
u 5 0
u 0 1
u 5 1
u 0 2
u 2 2
u 5 2
u 0 3
u 5 3
u 0 4
u 1 4
u 2 4
u 3 4
u 4 4
u 5 4
u 0 5
u 1 5
u 2 5
u 3 5 
u 4 5
u 5 5
u 4 3
f 1 1
f 1 2
f 1 3
f 2 1
f 2 3
f 3 1
f 3 2
f 3 3
f 4 1
f 4 2


