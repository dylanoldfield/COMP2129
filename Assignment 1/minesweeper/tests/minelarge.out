g 6 6
b 1 1
b 1 2
b 1 3
b 2 1
b 2 3
b 3 1
b 3 2
b 3 3
b 4 1
b 4 2
+------+
|******|
|******|
|******|
|******|
|******|
|******|
+------+
u 0 0
+------+
|1*****|
|******|
|******|
|******|
|******|
|******|
+------+
u 1 0
+------+
|12****|
|******|
|******|
|******|
|******|
|******|
+------+
u 2 0
+------+
|123***|
|******|
|******|
|******|
|******|
|******|
+------+
u 3 0
+------+
|1233**|
|******|
|******|
|******|
|******|
|******|
+------+
u 4 0
+------+
|12332*|
|******|
|******|
|******|
|******|
|******|
+------+
u 5 0
+------+
|123321|
|******|
|******|
|******|
|******|
|******|
+------+
u 0 1
+------+
|123321|
|2*****|
|******|
|******|
|******|
|******|
+------+
u 5 1
+------+
|123321|
|2****2|
|******|
|******|
|******|
|******|
+------+
u 0 2
+------+
|123321|
|2****2|
|3*****|
|******|
|******|
|******|
+------+
u 2 2
+------+
|123321|
|2****2|
|3*8***|
|******|
|******|
|******|
+------+
u 5 2
+------+
|123321|
|2****2|
|3*8**2|
|******|
|******|
|******|
+------+
u 0 3
+------+
|123321|
|2****2|
|3*8**2|
|2*****|
|******|
|******|
+------+
u 5 3
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|******|
|******|
+------+
u 0 4
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|1*****|
|******|
+------+
u 1 4
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|12****|
|******|
+------+
u 2 4
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123***|
|******|
+------+
u 3 4
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|1232**|
|******|
+------+
u 4 4
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|12321*|
|******|
+------+
u 5 4
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|******|
+------+
u 0 5
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|0*****|
+------+
u 1 5
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|00****|
+------+
u 2 5
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|000***|
+------+
u 3 5
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|0000**|
+------+
u 4 5
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|00000*|
+------+
u 5 5
+------+
|123321|
|2****2|
|3*8**2|
|2****1|
|123210|
|000000|
+------+
u 4 3
+------+
|123321|
|2****2|
|3*8**2|
|2***31|
|123210|
|000000|
+------+
f 1 1
+------+
|123321|
|2f***2|
|3*8**2|
|2***31|
|123210|
|000000|
+------+
f 1 2
+------+
|123321|
|2f***2|
|3f8**2|
|2***31|
|123210|
|000000|
+------+
f 1 3
+------+
|123321|
|2f***2|
|3f8**2|
|2f**31|
|123210|
|000000|
+------+
f 2 1
+------+
|123321|
|2ff**2|
|3f8**2|
|2f**31|
|123210|
|000000|
+------+
f 2 3
+------+
|123321|
|2ff**2|
|3f8**2|
|2ff*31|
|123210|
|000000|
+------+
f 3 1
+------+
|123321|
|2fff*2|
|3f8**2|
|2ff*31|
|123210|
|000000|
+------+
f 3 2
+------+
|123321|
|2fff*2|
|3f8f*2|
|2ff*31|
|123210|
|000000|
+------+
f 3 3
+------+
|123321|
|2fff*2|
|3f8f*2|
|2fff31|
|123210|
|000000|
+------+
f 4 1
+------+
|123321|
|2ffff2|
|3f8f*2|
|2fff31|
|123210|
|000000|
+------+
f 4 2
+------+
|123321|
|2ffff2|
|3f8ff2|
|2fff31|
|123210|
|000000|
+------+
won
